read-the-docs-2018-stats.rst

```diff
@@ -0,0 +1,107 @@
+.. post:: Jan 7, 2019
+   :tags: stats, year-in-review
+
+Read the Docs 2018 Stats
+========================
+
+2018 was another good year for Read the Docs.
+We've settled into a sustainability model that is working for us,
+and have a team of 5 folks working full-time on the project.
+
+Here are our stats for the past year,
+which we've published for the past 6 years.
+This is part of our effort to be transparent in our organization,
+as well as our source code.
+
+.. note:: 
+
+	You can always see our stats for the last `30 days`_. 
+
+	Our posts from 2013_, 2014_, 2015_, 2016_, and 2017_ are also available.
+
+.. _30 days: http://www.seethestats.com/site/readthedocs.org
+.. _2013: https://blog.readthedocs.com/read-the-docs-2013-stats/
+.. _2014: https://blog.readthedocs.com/read-the-docs-2014-stats/
+.. _2015: https://blog.readthedocs.com/read-the-docs-2015-stats/
+.. _2016: https://blog.readthedocs.com/read-the-docs-2016-stats/
+.. _2017: https://blog.readthedocs.com/read-the-docs-2017-stats/
+
+Page Views
+----------
+
+Our stats:
+
+* 400 Million Page Views (up from 338M)
+* 77 Million Unique Visitors (up from 75M)
+
+.. From Google Analytics
+
+These numbers are not as accurate this year as in the past,
+because we have :doc:`implemented Do Not Track support </do-not-track>` to our Google Analytics setup.
+So we're losing a non-trivial number of users in that 
+
+We however remain one of the `largest sites`_ on the internet.
+
+.. _largest sites: http://www.alexa.com/siteinfo/readthedocs.io
+
+Site Stats
+----------
+
+The stats, in total numbers:
+
+* 100,000 projects (up from 77k)
+* 155,000 users (up from 103k)
+
+We have been battling spam quite heavily again this year,
+so these numbers might be a bit skewed.
+
+.. Project.objects.count()
+.. User.objects.count()
+
+Community
+---------
+
+This year, we had:
+
+* 28 `people`_ who committed code to the main repository (up from 23)
+* 3774 `commits`_ (up from 1058)
+* 797 `issues`_ - 189 open, 608 closed (up from 513)
+
+.. git rev-list --count --all --after="2017-12-31" --before="2019-01-01"
+
+.. _people: https://github.com/rtfd/readthedocs.org/graphs/contributors?from=2018-01-01&to=2018-12-31&type=c
+.. _commits: https://github.com/rtfd/readthedocs.org/commits/master
+.. _issues: https://github.com/rtfd/readthedocs.org/issues?utf8=%E2%9C%93&q=is%3Aissue++created%3A2018-01-01..2019-01-01+
+
+Funding
+-------
+
+* $400,000 in revenue (up from $250,000)
+    * $300,000 from advertising
```

Comment:
Do you think it's worth mentioning the impact that ad blockers had on our ad revenue over the year? While it's a bit fuzzy, I think we could put together a pretty good estimate.

Reply:
Yea, could be interesting. Definitely worth mentioning and linking to, at least. But if we have approx. numbers that would be 👍

Reply:
I updated it with percentages. That seems probably good enough, but we could also just do `(Q3 + Q4 ad revenue) * .15` to get a number probably.

Reply:
15% is pretty much right on the money [based on our stats](https://docs.readthedocs.io/en/latest/advertising/ad-blocking.html#statistics-and-data).

The only thing I'd say is to not use the word "lost" when talking about ads being blocked. Some folks especially in the ad blocker communities don't like that. I'd phrase it more along the lines of "For the second half of the year, our ad revenue was down about 15%".